keyboards/keychron/k2/rgb/v2/ansi/info.json

```
@@ -0,0 +1,269 @@
+{
+    "keyboard_name": "Keychron K2v2 RGB ANSI",
+    "manufacturer": "Keychron",
+    "url": "https://www.keychron.com/products/keychron-k2-wireless-mechanical-keyboard",
+    "maintainer": "ITNerd69",
+    "usb": {
+        "pid": "0xFE02",
+		"vid": "0x0001",
```

```suggestion
		"vid": "0x3434",
```
see https://github.com/SonixQMK/qmk_firmware/issues/152

---

Removed